test/e2e/storage/csi_mock_volume.go

```diff
@@ -1301,6 +1338,99 @@ var _ = utils.SIGDescribe("CSI mock volume", func() {
 			})
 		}
 	})
+
+	// These tests *only* work on a cluster which has the CSIVolumeFSGroupPolicy feature enabled.
+	ginkgo.Context("CSI FSGroupPolicy", func() {
```

Comment:
Can you add `[LinuxOnly]` to this since fsgroup is not supported on windows

Reply:
This has been included.